src/test/resources/fixtures/rest/accountactivity/webhook.json

```diff
@@ -0,0 +1,6 @@
+{
+  "created_timestamp": "2020-06-22 15:19:10 +0000",
```

Comment:
Yeah, I think we can easily add a parser for this :)
Although probably it should be an `Instant`, but rather a `ZonedDateTime` or similar..?

Reply:
Indeed, I used `ZonedDateTime`